libs/ledger-live-common/src/wallet-api/react.ts

```diff
@@ -795,7 +795,7 @@ export function useCategories(): Categories {
     () => all.filter(m => ["complete", "searchable"].includes(m.visibility)),
     [all],
   );
-  const { categories, manifestsByCategories } = useCategoriesRaw(searchable);
+  const { categories, manifestsByCategories } = useCategoriesRaw(complete);
```

Comment:
Why do we change this part ?
It doesn't seem necessary but maybe I'm missing something

Reply:
`categories` contains all the categories of the manifests inside `complete` 
-> We only display the categories of complete apps 

LLM base the displayed manifests inside the categories on `manifestsByCategories`

Reply:
Okay but LLM uses `categories.searchable` instead of `categories.manifests.complete` like LLD

And now you put only the complete manifests in `categories` so `categories.searchable` doesn't make a lot of sense while on LLD it makes sense to me